Source/Dafny/Compiler-Csharp.cs

```diff
@@ -2409,7 +2410,7 @@ private class CSharpCompilationResult
       // * CS0436 is about types in source files that conflict with imported types (caused by
       //   dynamically-generated types like Tuple0 that aren't part of the runtime, which are
       //   often in pre-compiled Dafny DLLs)
-      cp.CompilerOptions = "/debug /nowarn:0164 /nowarn:0219 /nowarn:1717 /nowarn:0162 /nowarn:0168 /nowarn:0436";
+      cp.CompilerOptions = "/debug /nowarn:0164 /nowarn:0219 /nowarn:1717 /nowarn:0162 /nowarn:0168 /nowarn:0436 /nowarn:0183";
```

Comment:
Update `EmitHeader()` to reflect the new no-warn.

Reply:
Done

Reply:
Done